.github/workflows/pull_request.yml

```diff
@@ -0,0 +1,53 @@
+# CI stages to execute against Pull Requests
+name: Robottelo - CI
+
+on:
+  pull_request:
+    types: ["opened", "synchronize", "reopened"]
+
+env:
+    PYCURL_SSL_LIBRARY: openssl
+
+jobs:
+  codechecks:
+    name: Code Quality
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: [3.8, 3.9]
+    steps:
+      - name: Checkout Airgun
+        uses: actions/checkout@v2
+
+      - name: Set Up Python-${{ matrix.python-version }}
+        uses: actions/setup-python@v2
+        with:
+          python-version: ${{ matrix.python-version }}
+
+      - name: Install Dependencies
+        run: |
+          sudo apt update
```

Comment:
Looks like this is copy-pasta from robottelo GHA, and this block can be simplified a lot.

I think its good to keep the block around to continue testing airgun installation under pip, but could be simplified all the way down to:

```
sudo apt update
pip install -U pip
pip install -Ur requirements.txt
```

Reply:
For a bit more thorough coverage, you could also try installing from `requirements-optional.txt`, or install as `pip install .`, since requirements for airgun are really included in `setup.py`.

Reply:
done